README.md

```diff
@@ -1,14 +1,64 @@
+
+[sales@syncfusion.com](mailto:sales@syncfusion.com?Subject=Syncfusion Blazor Chips - GitHUb) | [www.syncfusion.com](https://www.syncfusion.com?utm_source=github&utm_medium=listing&utm_campaign=blazor-chips-github-samples) | Toll Free: 1-888-9 DOTNET
```

Comment:
remove this line. 
you've already added it here
![image](https://user-images.githubusercontent.com/61148953/167833500-366cb215-b676-4cb0-9e81-88f42dea4439.png)

Reply:
resolved